ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -73,7 +73,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class Cli implements Closeable, AutoCloseable {
+public class Cli implements Closeable {
```

Comment:
Would it make sense to implement `AutoCloseable` instead? @confluentinc/ksql  WDYT?

Reply:
Don't think it much matters.